docs/pages/steps/jira_add_comment.md

```diff
@@ -10,10 +10,7 @@ folder: steps
 
```

Comment:
Please write a single document with some examples to cover about global, environment and non-env variables. So that we can avoid writing same examples in all the steps. But we can refer the new document in every step. 

And we can concentrate on writing multiple examples specific to this step.